jupyterhub-training-notebooks/basics.ipynb

```diff
@@ -0,0 +1,185 @@
+    "\n",
+    "## Navigating Jupyterlab on Jupyterhub \n",
+    "When you first log in you will have the JupyterLab interface by default. This should look like the image below. \n",
+    "\n",
+    "This consists of a main work area containing the _launcher_ [1]. As you open notebooks, files and terminals tabs will open in this area. There is a collapsible left sidebar containing a file browser, the list of running kernels and terminals, the command palette, the notebook cell tools inspector, and the tabs list [3]. Don't worry about this left sidebar too much. Along the top there is also a menu bar [3].\n",
+    "\n",
+    "<img src=\"../images/jupyterlab_with_launcher.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "For more information on this see the documentation [here](https://jupyterlab.readthedocs.io/en/latest/user/interface.html)\n",
+    "\n",
+    "When you open a notebook a new tab will open in the main work area. The notebook should consist of a mix of text and greyed out boxes called cells. \n",
+    "\n",
+    "<img src=\"../images/jupyterlab_with_notebook_open.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "\n",
+    "## Running a cell \n",
+    "By defualt you should be in Jupyterlab when you log in. There are several ways of running a cell in this format. The quickest way is to select the cell you would like to run and click `Shift`+`Enter`\n",
+    "\n",
+    "<img src=\"../images/select_cell.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Or select the cell you would like to run then select _Run -> Run Selected Cells_ from the menu bar. \n",
+    "\n",
+    "<img src=\"../images/run_selected_cells.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Or select the cell you would like to run and click the play button at the top of the main work area.\n",
+    "\n",
+    "<img src=\"../images/play_cell.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "Pick your favourite method and have a go at running the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "echo \"Hello World! Congratulations, you got your first cell to run\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to tell when things are running \n",
+    "When things are running a `*` symbol will appear in the square brackets to the left of the cell. \n",
+    "\n",
+    "<img src=\"../images/cell_running.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
+    "\n",
+    "When this has finished running it will change to a number.\n",
+    "\n",
+    "<img src=\"../images/cell_completed_running.png\" alt=\"Drawing\" style=\"width: 700px;\"/>\n",
```

Comment:
the  images cell_running.png and cell_completed_running.png contain text as well as screenshots of the running/completed cell.   The text looks is a rendering of the markdown  in this cell, so I guess it was included by mistake?